src/content/wiki/build.mdx

```diff
@@ -1,7 +1,6 @@
 ---
 title: Build
 description: Instructions for building the Lua Language Server from source.
-getting-started: true
```

Comment:
I think keeping the build instructions in the getting started section would be good as it can not only be used for development but also simply for users who prefer/need building from source.

Reply:
Yeah, I totally forgot about that group of users, restoring.